src/components/Sections/GoogleMapsSection/index.tsx

```diff
@@ -0,0 +1,179 @@
+    stylers: [{ color: "#ffffff" }, { lightness: 18 }],
+  },
+  {
+    featureType: "road.local",
+    elementType: "geometry",
+    stylers: [{ color: "#ffffff" }, { lightness: 16 }],
+  },
+  {
+    featureType: "poi",
+    elementType: "geometry",
+    stylers: [{ color: "#f5f5f5" }, { lightness: 21 }],
+  },
+  {
+    featureType: "poi.park",
+    elementType: "geometry",
+    stylers: [{ color: "#dedede" }, { lightness: 21 }],
+  },
+  {
+    elementType: "labels.text.stroke",
+    stylers: [{ visibility: "on" }, { color: "#ffffff" }, { lightness: 16 }],
+  },
+  {
+    elementType: "labels.text.fill",
+    stylers: [{ saturation: 36 }, { color: "#333333" }, { lightness: 40 }],
+  },
+  { elementType: "labels.icon", stylers: [{ visibility: "off" }] },
+  {
+    featureType: "transit",
+    elementType: "geometry",
+    stylers: [{ color: "#f2f2f2" }, { lightness: 19 }],
+  },
+  {
+    featureType: "administrative",
+    elementType: "geometry.fill",
+    stylers: [{ color: "#fefefe" }, { lightness: 20 }],
+  },
+  {
+    featureType: "administrative",
+    elementType: "geometry.stroke",
+    stylers: [{ color: "#fefefe" }, { lightness: 17 }, { weight: 1.2 }],
+  },
+] as google.maps.MapTypeStyle[];
+
+@Component({
+  name: "GoogleMapsSection",
+})
+class GoogleMapsSection extends BaseComponent<GoogleMapsSectionProps> {
+  map: google.maps.Map | null = null;
+
+  @Prop() title: GoogleMapsSectionProps["title"];
+  @Prop({ required: false }) apikey!: GoogleMapsSectionProps["apikey"];
+  @Prop({ required: false })
+  startLocation!: GoogleMapsSectionProps["startLocation"];
+  @Prop({ required: false, default: 15 }) zoom!: GoogleMapsSectionProps["zoom"];
+  @Prop({ required: false, default: "en" })
+  language!: GoogleMapsSectionProps["language"];
+  @Prop({ required: false }) locations!: GoogleMapsSectionProps["locations"];
+  @Prop({ required: false, default: "Contact Us" })
+  buttonLabel!: GoogleMapsSectionProps["buttonLabel"];
+  @Prop({ required: true })
+  handleButtonClick!: GoogleMapsSectionProps["handleButtonClick"];
+
+  placeMarkers(map: google.maps.Map, ...locations: MapsLocation[]): void {
+    const renderDescriptionBox = (location: MapsLocation): Node => {
+      const template = `<h2 class="font-bold">${location.name}</h2>
+        <p>${location.street}</p>
+        ${location.description ? "<p>" + location.description + "</p>" : ""}
+        <p>${location.city}</p>
+        <button class="bg-black text-white hover:bg-gray-300 hover:text-black focus:outline-none p-2 my-4 w-auto overflow-hidden cursor-pointer font-light text-sm">
+        ${this.buttonLabel}
+        </button>`;
+      const div = document.createElement("div");
+      div.classList.add("w-32", "text-sm");
+      div.innerHTML = template;
+      div.querySelector("button")?.addEventListener("click", event => {
+        this.handleButtonClick(event, location);
+      });
+      return div;
+    };
+
+    locations.map((location: MapsLocation) => {
```

Comment:
Since we don't work with the `.map()`-result, we should use `.forEach()` in this case.
```suggestion
    locations.forEach((location: MapsLocation) => {
```